app/jobs/release_object_job.rb

```diff
@@ -0,0 +1,112 @@
+##
+# Job to add release and then release a Dor Object
+class ReleaseObjectJob < GenericJob
+  queue_as :release_object
+
+  attr_reader :manage_release, :pids, :webauth
+  ##
+  # This is a shameless green approach to a job that calls release from dor
+  # services app and then kicks off release WF.
+  # @param [Integer] bulk_action_id GlobalID for a BulkAction object
+  # @param [Hash] params additional parameters that an Argo job may need
+  # @option params [Array] :pids required list of pids
+  # @option params [Hash] :manage_release required Hash of release options
+  # @option manage_release [String] :to required release to target
+  # @option manage_release [String] :who required username of releaser
+  # @option manage_release [String] :what required type of release (self, collection)
+  # @option manage_release [String] :tag required (true, false)
+  # @option webauth [Hash] required for permissions check
+  def perform(bulk_action_id, params)
+    @manage_release = params[:manage_release]
+    @webauth = OpenStruct.new params[:webauth]
+    @pids = params[:pids]
+    File.open(bulk_action.log_name, 'w') do |log|
+      log.puts("#{Time.current} Starting ReleaseObjectJob for BulkAction #{bulk_action_id}")
+      update_druid_count
+
+      pids.each do |current_druid|
+        next unless can_manage?(current_druid)
```

Comment:
What about adding a log message here, so we don't "silently" pass on to the next one in case the user by mistake tried to operate on a druid they can't manage?